provider/cmd/pulumi-resource-eks/schema.json

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
Just to confirm: this isn't currently used, right?

Reply:
This file is generated and we check it in like we do in other repos. It is actually used. In the `Makefile` there's a `schema` target that generates this file and the `build_python` and `build_dotnet` targets depend on the `schema` target and do actually read-in the generated `schema.json` in order to generate the code.

https://github.com/pulumi/pulumi-eks/blob/b69072bb65b74f5a6c138031c0af5767980bb4d8/Makefile#L30-L32